### DISCUSSION TITLE
I need help with a custom ComfyUI node set

### DISCUSSION THREAD

**Discussion by Lorenzoidal** [community/community#131648] [Programming Help]
### Body

Hello, I've been desperately trying to run StyleGAN2 and StyleGAN3 models locally for years, and recently I stumbled upon the repository by spacepxl that will load StyleGAN *.pkl files in ComfyUI!

https://github.com/spacepxl/ComfyUI-StyleGan/tree/main

Only problem is that I have no way to contact them, and I've tried everything to fix the issue I'm having. The nodes I need will install and load just fine, but when I run them I get the error:

"Error occurred when executing LoadStyleGAN:

No module named 'dnnlib.tflib'"

I was fairly sure I had gotten the right *.whl file so that I could pip install it, but even after finding it and installing it I still get the error. :(

Can anyone please help me, or tell me how to contact the developer?

TYVM in advance!

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @Lorenzoidal, thanks for being a part of the GitHub Community!

I see that they commented on [this issue](https://github.com/spacepxl/ComfyUI-StyleGan/issues/5) you left on the project's repository. 

For future reference, you can always open an issue or PR in a repo like you did, or contact users on their website / social media if there are links in their profile. There is currently no way to contact users directly on GitHub.

**Reply by Lorenzoidal** [reply]
My apologies, I followed a link that said "Ask me a question", and thought I'd stumbled onto a way to DM someone here.
